pkg/ovnnorthd/deployment.go

```diff
@@ -0,0 +1,125 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ovnnorthd
+
+import (
+	"github.com/openstack-k8s-operators/lib-common/modules/common"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/affinity"
+	"github.com/openstack-k8s-operators/lib-common/modules/common/env"
+	ovnv1 "github.com/openstack-k8s-operators/ovn-operator/api/v1alpha1"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// ServiceCommand -
+	ServiceCommand = "/usr/local/bin/kolla_set_configs && /usr/local/bin/kolla_start"
+)
+
+// Deployment func
+func Deployment(
+	instance *ovnv1.OVNNorthd,
+	configHash string,
+	labels map[string]string,
+) *appsv1.Deployment {
```

Comment:
Can we have this use the `lib-common` Deployment wrapper to maintain consistency with other operators?

Reply:
Using NewDeployment method in controllers which returns Deployment wrapper from lib-common